ngs_mapper/log.py

```diff
@@ -18,6 +18,7 @@ def setup_logger( name, config ):
     }
     #logging.config.dictConfig( config )
     logconfig.from_dict(config)
+    logging.getLogger("sh").setLevel(logging.WARNING)
```

Comment:
I guess if this works that is fine, but it seems like we should be modifying the config dictionary instead?